jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh

```diff
@@ -28,7 +28,7 @@ terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.0.1"
 
 welder_server="us.gcr.io/broad-dsp-gcr-public/welder-server:f411762"
 openidc_proxy="broadinstitute/openidc-proxy:2.3.1_2"
-anvil_rstudio_bioconductor="us.gcr.io/anvil-gcr-public/anvil-rstudio-bioconductor:3.13.0"
+anvil_rstudio_bioconductor="us.gcr.io/anvil-gcr-public/anvil-rstudio-bioconductor:3.13.2"
```

Comment:
can you update welder hash that's cached as well? it should be `97a2d72` now

Reply:
+1, will do when I regenerate the image for https://github.com/DataBiosphere/leonardo/pull/2271